### DISCUSSION TITLE
Keyboard click sounds play at very loud volumes

### DISCUSSION THREAD

**Discussion by peppy** [blinksh/blink#1936] [Support]
### Checklist

- [x] Using latest testflight build
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

17.2.0 build 855
iPhone 15 pro max

### Describe the bug

When turning on smart keys + click sounds, the click sounds will occasionally play at very loud volumes, blasting the iPhone's speaker.

This usually happens after pressing a key on the native keyboard first, followed by a smart key.

It does not reproduce in video, so I believe blink must be adjusting the volume or playing the same back in a weird way.

This happens about 50% of the time, and has been an issue for some time now.

I also just noticed the same thing happens for the notification bell.

https://github.com/blinksh/blink/assets/191335/fe5782a5-e303-4037-a023-807f5a942e5c

**Comment by carloscabanero** [comment]
Hi! I'm not sure this is under our control as we use `AudioServicesPlaySystemSound`. But have you tried to configure the Alerts and Ringtones in Settings? https://stackoverflow.com/questions/10855874/audioservicesplaysystemsound-volume. If you don't find a way and it is bothering you, it is possible to disable keyboard sounds inside Blink at Config > Smart Keys > Click sounds.

**Reply by peppy** [reply]
Yes I have disabled sounds for the safety of my hardware.

Is it safe to say you couldn't reproduce? I have across multiple devices and if you watched the video, I'd be concerned for users' hardware at this point 😅

My alert volume is set very low::

![image](https://github.com/blinksh/blink/assets/191335/3cf51b05-4708-43f7-a83d-4079bd27c11c)

**Comment by carloscabanero** [comment]
It is 💯 reproducible. I will try to use one of the regular sounds as maybe this one in particular has a special random behavior 😅

Thanks for the heads up. Moving back to issues.

**Reply by carloscabanero** [reply]
I can confirm the fix and it will be out on v17.2.0. https://github.com/blinksh/blink/issues/1937
